tests/test_exiftool_direct.py

```
@@ -0,0 +1,79 @@
+        
+        # Lire
+        result = subprocess.run([
+            "exiftool", "-json", str(media_path)
+        ], capture_output=True, text=True, check=True)
+        data = json.loads(result.stdout)[0]
+        print(f"   Keywords après ajout initial: {data.get('Keywords', 'AUCUN')}")
+        
+        # 2. Essayer -=, +=
+        print("\n2. Test avec -= puis +=...")
+        try:
+            result = subprocess.run([
```

Ruff (0.12.2)

`subprocess` call: check for execution of untrusted input

(S603)